.build/cassandra-build-deps-template.xml

```diff
@@ -123,5 +123,9 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+    <dependency>
+      <groupId>com.carrotsearch.randomizedtesting</groupId>
+      <artifactId>randomizedtesting-runner</artifactId>
+    </dependency>
```

Comment:
New dependencies should be approved in the dev mail list. I don't think that anyone is going to oppose to it, but we should send an email about it.

Reply:
I think the possible objection is that we already have QuickTheories in use in a few places. I responded to David on the ML..